l3build-install.lua

```diff
@@ -97,7 +97,7 @@ function uninstall()
   if errorlevel ~= 0 then return errorlevel end
   -- Finally, clean up special locations
   for _,location in ipairs(tdslocations) do
-    local path,glob = splitpath(location)
+    local path = splitpath(location)
     errorlevel = zapdir(path)
     if errorlevel ~= 0 then return errorlevel end
   end
```

Comment:
`glob` is unused in the block

Reply:
True but it is returned ... to me, a function returning two values needs two places to put them, used or not.

Reply:
The lua 5.3 reference reads
"Before the assignment, the list of values is adjusted to the length of the list of variables. If there are more values than needed, the excess values are thrown away. If there are fewer values than needed, the list is extended with as many nil's as needed."

Reply:
Sure, I know it works, I just meant I find it a bit odd.

Reply:
In l3build-ctan.lua line 42, there is the same scheme whereas in l3build-tagging.lua line 52 there is a
`local path = dirname(blablabla)`
I change to the latter in both files.